clippy_lints/src/raw_strings.rs

```diff
@@ -96,8 +96,9 @@ impl EarlyLintPass for RawStrings {
                         );
                     },
                 );
-
-                return;
+                if !matches!(cx.get_lint_level(NEEDLESS_RAW_STRINGS), rustc_lint::Allow) {
```

Comment:
There's a clippy wrapper called `is_lint_allowed`, it should work here with the expr's HirId, I think.

Reply:
It looks that `is_lint_allowed` need a `LateContext` argument, but we have only `EarlyContext` here.

Reply:
oh~ right, my bad